lib/helpers/createSpecification.js

```diff
@@ -18,9 +18,16 @@ function createSpecification(definition) {
     'parameters',
     'securityDefinitions',
   ];
-  const v3 = [...v2, 'components'];
+  let v3 = [...v2, 'components'];
```

Comment:
as noted in the comments, do not eagerly share the parser. Also this mutation is not very nice